packages/components/checkbox/src/CheckboxIndicator.tsx

```diff
@@ -10,7 +10,7 @@ export const CheckboxIndicator = forwardRef<HTMLSpanElement, CheckboxIndicatorPr
   (props, ref) => (
     <CheckboxIndicatorPrimitive
       ref={ref}
-      className="text-surface flex items-center justify-center"
+      className="text-surface flex h-full w-full items-center justify-center"
```

Comment:
i think this classes should be placed in xxxxx.styles.tsx to decouple the view from the logic

Reply:
I think it is one of the strengths of Tailwind to be able to apply some styles on the markup. It might not be necessary to always split the two ?

Reply:
I agree with @Powerplex but we can discuss it later on